runtime/src/accounts_db.rs

```diff
@@ -128,14 +128,14 @@ const CACHE_VIRTUAL_STORED_SIZE: StoredSize = 0;
 pub const ACCOUNTS_DB_CONFIG_FOR_TESTING: AccountsDbConfig = AccountsDbConfig {
     index: Some(ACCOUNTS_INDEX_CONFIG_FOR_TESTING),
     accounts_hash_cache_path: None,
-    filler_account_count: None,
+    filler_accounts_config: FillerAccountsConfig { count: 0, size: 0 },
```

Comment:
```suggestion
    filler_accounts_config: FillerAccountsConfig::default(),
```

Reply:
I did this originally but I got the following error:
![image](https://user-images.githubusercontent.com/151543/161583429-ad05f395-5c8d-476e-9fc0-2a63c6468979.png)
Do you know a way to overcome this limitation?

Reply:
Ah, it's `const`, I see. What you have currently is sufficient then. No changes necessary!

Reply:
We can still dereplicode this.  I'll add a suggestion below